source/vibe/core/drivers/libasync.d

```diff
@@ -760,7 +761,7 @@ final class LibasyncManualEvent : ManualEvent {
 				auto evloop = getEventLoop();
 				shared AsyncSignal sig = cast(shared AsyncSignal) signal;
 				if (!sig.trigger(evloop))
-					throw new Exception(sig.error);
+					return; //throw new Exception(sig.error);
```

Comment:
In which situation could this happen ? Wouldn't an asset be better over a silent fail ?